drivers/overlay/ov_serf.go

```diff
@@ -146,6 +146,7 @@ func (d *driver) processQuery(q *serf.Query) {
 		return
 	}
 
+	logrus.Debugf("Sending peer query resp %v", fmt.Sprintf("%s %s %s", peerMac.String(), net.IP(peerIPMask).String(), vtep.String()))
```

Comment:
@aboch Made the change. 

`peerIPMask` is ok type `net.IPMask` and the default Stringer for `net.IPMask` is different from `net.IP`. That is why it was casted explicitly.